tests/test-components-make-base.yaml

```diff
@@ -0,0 +1,24 @@
+id: tco
+title: "Test Components Make Base Ontology"
+github_org: INCATools
+repo: ontology-development-kit
+release_artefacts: 
+  - full
+  - simple
+primary_release: full
+export_formats:
+  - owl
+  - json
+import_group:
+  products: 
+    - id: ro
+    - id: bfo
+components:
+  products:
+    - filename: pato.owl
+      base_iris:
+        - http://purl.obolibrary.org/obo/PATO_
+        - http://purl.obolibrary.org/obo/pato#
+      make_base: True
```

Comment:
Maybe I’m missing something, but shouldn’t there be a `source` parameter for the PATO component in this test case? The `make_base` parameter only has an effect in the `component-download-...` rule, which itself is only generated if there is an explicit `source`.

Reply:
Oooops, yeap this was too hastily done. Thanks for catching!